src/main/java/de/bas/deploymentmanager/logic/domain/stage/control/StageServiceImpl.java

```diff
@@ -19,15 +22,19 @@
 public class StageServiceImpl implements StageService {
     private final Logger log = LoggerFactory.getLogger(this.getClass());
 
+    private final ProjectRepository projectRepository;
     private final StageRepository stageRepository;
     private final AppRepository appRepository;
     private final HostRepository hostRepository;
+    private final ImageRepository imageRepository;
```

Comment:
Inject von ImageRepository ist nicht erlaubt an dieser Stelle. 
Die beiden befinden sich in Unterschiedlichen Domains 

Funktionalität sollte im LoadStageFlow gebündelt werden

Reply:
s.o. sind ja schon andere Repos mit drin. Ist aber wieder raus wie oben.

Reply:
Ja aber es sind nur Repository drin, die au dem gleichen Package kommen. Die Bereiche sollten so gut wie möglich getrennt von einander bleiben.